README.md

```diff
@@ -16,12 +16,9 @@ such as
 [`dprint-plugin-typescript`](https://github.com/dprint/dprint-plugin-typescript)
 for JavaScript / TypeScript.
 
-## Binaries
+## Usage
 
-The latest version of the plugin can be downloaded from the github releases page or from `plugins.dprint.dev`:
-```
-https://plugins.dprint.dev/malobre/vue-latest.wasm
```

Comment:
People should generally not use the `-latest.wasm` url because the dprint cli currently caches it indefinitely and the only practical way to update is to clear the entire cache. Instead it's better for users to use one of the versioned urls.

Also, see https://github.com/dprint/dprint/issues/451 for more details on the future of these `-latest.wasm` urls... they might be useful for global configuration.